cli-files/go-ls/cmd/root_test.go

```diff
@@ -0,0 +1,82 @@
+package cmd
+
+import (
+	"bytes"
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var TestTable = map[string]struct {
+	Args  []string
+	Want  *string
+	Error string
+}{
+	"No path provided": {
+		Args:  []string{},
+		Want:  StringToPointer("root.go\nroot_test.go"),
```

Comment:
In general we tend to add a trailing newline to the output of CLI tools, so I'd expect this `Want` to be `"root.go\nroot_test.go\n"` rather than `"root.go\nroot_test.go"` - this is because:
1. Some other tools only handle lines that end in trailing newlines
2. Terminals tend to display this slightly better

It has the side effect of also allowing you to remove a special case in your code!

Reply:
Ok makes sense, thanks!